released/packages/coq-switch/coq-switch.1.0.0/descr

```diff
@@ -0,0 +1,8 @@
+A plugin to imlement functionality similar `switch` operator in other
+languages.
```

Comment:
I think it would be clearer to say something along the lines "C-like switch statement syntax".
I was quite puzzled from the word "operator".
Maybe C-like is not the best parallel to draw, given the Coq public, but in any case I suggest you pick a concrete lange in the description.